lua/doom/modules/langs/lua/init.lua

```diff
@@ -13,15 +13,15 @@ lua.settings = {
   disable_lsp = false,
   --- Name of the language server
   --- @type string
-  lsp_name = "sumneko_lua",
+  lsp_name = "lua_ls",
```

Comment:
I've changed the default language server because `sumneko_lua` is not supported anymore and will be removed in nvim-lspconfig `0.2.0`.